iulianIachimov/src/main/java/dataCollectionTask/Models/User.java

```diff
@@ -1,44 +1,14 @@
 package dataCollectionTask.Models;
 
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Map;
 
-public class User {
-  public long id;
-  public String username;
-  public Map<String, Address> addresses;
-  private static long idCounter;
+public class User extends AbstractUser {
```

Comment:
where is the User attribute? What is the difference between this class and the parent one>?